beat/config.py

```diff
@@ -599,6 +599,8 @@ class WaveformFitConfig(Object):
         optional=True,
         help='Index to event from events list for reference time and data '
              'extraction. Default is 0 - always use the reference event.')
+    freqdomain_include = Bool.T(default=True, help='Flag to include spectrum into optimization.')
```

Comment:
you only need a single flag e.g. spectral_domain,
if thats true, spectra if not time-domain ...

Reply:
What if we don't want the time domain?

Reply:
then all spectral_domain flags will be true